frontend/src/components/phones/onboarding/RealPhoneSetup.tsx

```diff
@@ -315,3 +261,55 @@ const RealPhoneVerification = (props: RealPhoneVerificationProps) => {
     </div>
   );
 };
+
+type IntervalFunction = () => unknown | void;
+// See https://overreacted.io/making-setinterval-declarative-with-react-hooks/
+function useInterval(callback: IntervalFunction, delay: number) {
+  const savedCallback = useRef<IntervalFunction | null>(null);
+
+  // Remember the latest callback.
+  useEffect(() => {
+    savedCallback.current = callback;
+  }, [callback]);
+
+  // Set up the interval.
+  useEffect(() => {
+    function tick() {
+      if (savedCallback.current !== null) {
+        savedCallback.current();
+      }
+    }
+    if (delay !== null) {
+      const id = setInterval(tick, delay);
+      return () => clearInterval(id);
+    }
+  }, [delay]);
+}
+
+function getRemainingTime(verificationSentTime: Date): number {
+  const currentTime = new Date().getTime();
+  const elapsedTime = currentTime - verificationSentTime.getTime();
+  const remainingTime = 5 * 60 * 1000 - elapsedTime;
+  return remainingTime;
+}
+
+function formatPhone(phoneNumber: string) {
+  // Bug: Any country code with two characters will break
+  // Return in +1 (111) 111-1111 format
+  return `${phoneNumber.substring(0, 2)} (${phoneNumber.substring(
+    2,
+    5
+  )}) ${phoneNumber.substring(5, 8)}-${phoneNumber.substring(8, 12)}`;
+}
+
+function getPhoneWithMostRecentlySentVerificationCode(
+  phones: UnverifiedPhone[]
+) {
+  const mostRecentVerifiedPhone = [...phones].sort((a, b) => {
+    const sendDateA = parseDate(a.verification_sent_date).getTime();
+    const sendDateB = parseDate(b.verification_sent_date).getTime();
+    return sendDateA - sendDateB;
+  });
+
+  return mostRecentVerifiedPhone[0];
+}
```

Comment:
question (non-blocking): Looks like Luke already asked about this. All we did was move down right?

Reply:
Yep sorry, see https://github.com/mozilla/fx-private-relay/pull/2298#discussion_r937447830.